daemon/volumes.go

```diff
@@ -220,11 +219,9 @@ func (s *volumeStore) Decrement(v volume.Volume) {
 	logrus.Debugf("Decrementing volume reference: driver %s, name %s", v.DriverName(), v.Name())
 
 	vc, exists := s.vols[v.Name()]
-	if !exists {
```

Comment:
Sorry, I've taken back my lgtm after thinking more, this is almost the same block as the one above in `Increment` and I live err check before so for me the only things to do here is to remove the `return` as you did in `Increment`